notebooks/jupyter_dev_example.ipynb

```diff
@@ -252,8 +252,16 @@
    "source": [
     "index = \"local_development_example_index_withentity\" # You can change this to something else if you'd like\n",
     "\n",
-    "os_client_args = {\n",
-    "        \"hosts\": [{\"host\": \"localhost\", \"port\": 9200}],\n",
+    "if os.path.exists(\"/app/work/bind_dir\"):\n",
```

Comment:
I think the canonical way to check is to look for the file /.dockerenv but any of these methods can be fooled.